src/lib/react/useRoomConnection.ts

```diff
@@ -311,6 +319,18 @@ function reducer(state: RoomConnectionState, action: RoomConnectionEvents): Room
                 ...state,
                 screenshares: state.screenshares.filter((ss) => !ss.isLocal),
             };
+        case "LOCAL_CAMERA_ENABLED":
+            if (!state.localParticipant) return state;
```

Comment:
When can `state.localParticipant` be false? (Before joining the room I guess? And is it expected to not do anything when client clicks the toggle mic btn?)

Reply:
That is a good question. Probably never in practice, but according to our types it can be, so this is needed in order to not get a type error on the lines below